graphs/heat_map.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+from datetime import datetime, timedelta
+import argparse
+import itertools
+import numpy as np
+import yaml
+from yaml import CLoader as Loader
+from plotly.graph_objs import Figure, Layout, Scatter
+import plotly.express as px
+
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+
+parser = argparse.ArgumentParser(description='COVID-19 visualization for Slovakia')
+parser.add_argument('simulated',
+                    metavar='simulated',
+                    type=str,
+                    help=f"YAML file with simulation results")
+args = parser.parse_args()
+
+with open(args.simulated, 'r') as stream:
+    try:
+        data = yaml.load(stream, Loader=Loader)
+    except yaml.YAMLError as exc:
+        raise exc
+
+    best_errors = {}
+    for group in data:
+        if ("results" not in group) and ("result_abbrev" not in group):
+            continue
```

Comment:
Aj takéto sa stáva? Potom máme niekde bug, nie? Ja by som sem dal natvrdo assert, lebo to je naša chyba a zvyšuje to šancu zlých výsledkov, ak si to nevšimneme.

Reply:
Ako som kopíroval pôvodný kod z iného súboru, toto tam bolo a tiež som si myslel, že sa to stáva. Potom som zistil, že existuje aj result_abbrev a pochopil, že to tam je zrejme kvôli tomu ale nebol som si istý a nakoľko je dát málo nevedel som či sa to nemôže stať a nechal to tam ošetrené. Pridám assert ak je to potrebné?

Reply:
Môžeme mať `assert  ("results" in group) or ("result_abbrev" in group)`

Reply:
Všimol som si `if "results" not in group: continue` v kóde. To je podľa mňa ešte pozostatok toho, keď som ešte nepridal result_abbrev.

Reply:
> Všimol som si `if "results" not in group: continue` v kóde scatter_plot. To je podľa mňa ešte pozostatok toho, keď som ešte nepridal result_abbrev.

To som myslel :+1:

Reply:
Hotovo